docs/locale.md

```diff
@@ -0,0 +1,25 @@
+---
+sidebar_position: 9
+---
+
+# Locale
+
+**What are the languages supported by Topper?**
+
+Topper supports English and Brazilian Portuguese.
+
+**How to set a specific locale?**
+
+Without any param passed, the widget will try to infer the user locale from the browser and if the browser locale is not supported it will default to English.
```

Comment:
By default, Topper will use the locale set on the user's browser. If that locale is not supported it will default to English.

To set a specific locale, you can set it as query parameter such as: `https://app.topperpay.com/?bt=<bootstrap token>&locale=en`. The supported values are `en` and `pt`. 

When the user user authenticates in Topper, the locale will be set accordingly with the user preferences.